Pipfile

```diff
@@ -4,7 +4,9 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pytest = "*"
+black = "20.8b1"
+pre-commit = "2.7.1"
+pytest = "6.0.1"
```

Comment:
What did you do to get these version numbers? I'm glad they're there -- but I'm curious why they didn't get generated when I ran `pipenv install` commands.